src/main/java/virtual_pet/VirtualPetApplication.java

```diff
@@ -1,9 +1,100 @@
 package virtual_pet;
 
+import java.util.Scanner;
+
 public class VirtualPetApplication {
+    public static Virtualpet Rick = new Virtualpet(3, 3, 3, 3);
 
     public static void main(String[] args) {
-        //Interact with a VirtualPet object in this method
+
+        System.out.println("Welcome to Virtual pet!");
+        System.out.println("try to keep the pet happy so he doesn't run away!");
+        System.out.println("Will run away when any of his needs reach 10");
+        displayCurrentNeeds();
+        displayGameObjective();
+
+
+            while (Rick.getHungerLevel() <= 10 && Rick.getThirstLevel() <= 10 && Rick.getBoredomLevel() <= 10 && Rick.getBathroomBreak() <= 10) {
+                System.out.println("What would you like to do? ");
+                System.out.println("Want to feed, type 1");
+                System.out.println("Want to water, type 2");
+                System.out.println("To play with pet, type 3");
+                System.out.println("To give a bathroom break, type 4");
+                System.out.println("To do nothing at this time, type 5");
+                System.out.println("To end the game, type 6");
+
+                Scanner scanner1 = new Scanner(System.in);
+                int userSelection = Integer.valueOf(scanner1.next());
```

Comment:
Using a primitive int here would have allowed you to simplify your code where all you would have needed to do was code: int userSelection = scanner1.nextInt();